BDArmory/Weapons/Missiles/MissileLauncher.cs

```diff
@@ -1676,6 +1689,44 @@ public override void OnFixedUpdate()
                     UpdateGuidance();
                     CheckDetonationState(); // this needs to be after UpdateGuidance()
                     CheckDetonationDistance();
+
+                    if (missileCMRange > 0)
+                    {
+                        if (CMenabled)
+                        {
+                            if (missileCMInterval > 0 && (Time.time - missileCMTime) > missileCMInterval)
+                            {
+                                missileCMTime = Time.time;
+
+                                foreach (CMDropper dropper in missileCM)
+                                {
+                                    dropper.DropCM();
+                                }
+                            }
+                        }
+                        else if ((TargetPosition - vessel.CoM).sqrMagnitude < missileCMRange * missileCMRange)
+                        {
+                            var ECM = part.FindModuleImplementing<ModuleECMJammer>();
+                            if (ECM != null)
+                            {
+                                ECM.EnableJammer();
+                                CMenabled = true;
+                            }
+
+                            //missileCM = new List<CMDropper>();
+                            missileCM = part.FindModulesImplementing<CMDropper>();
+                            missileCMTime = Time.time;
+                            foreach (CMDropper dropper in missileCM)
+                            {
+                                if (dropper.cmType == CMDropper.CountermeasureTypes.Chaff)
```

Comment:
VCI is a single instance per vessel so it's unnecessary to call `UpdateVCI` multiple times. Also, the VCI check is already done when a chaff gets dropped in `CMDropper.DropChaff`, so this call is somewhat redundant.

Reply:
When the missile gets launched it becomes its own vessel, `UpdateVCI` is there to set the VCI to the new missile vessel's VCI, rather than the original vessel's VCI.